resources/content/db/changelog.xml

```diff
@@ -64,5 +64,7 @@
     <include file="db/core-057.xml"/>
     <include file="db/core-058.xml"/>
     <include file="db/core-059.xml"/>
+    <include file="db/core-060.xml"/>
```

Comment:
Guessing you know this already, but 60 is already on master, so I guess you need to rebase.